vector/src/main/java/im/vector/activity/VectorHomeActivity.java

```diff
@@ -487,11 +488,18 @@ private void showFloatingActionButton() {
 
     }
 
+    private void applyScreenshotSecurity() {
```

Comment:
Please move this method to RiotAppCompatActivity and set protected as visiblity to avoid code duplication